src/components/menu/GitBranchActionMenu.vue

```diff
@@ -24,6 +24,20 @@
           {{ $t('actions.git.newBranchFrom', { branch: branchName }) }}
         </q-item-section>
       </q-item>
+
+      <template v-if="onLocal">
```

Comment:
not convinced by the naming `onLocal`. Maybe something more specific ?

Reply:
I take the same name of Branch attributes to be consistent